src/Item/Item.service.ts

```diff
@@ -120,7 +120,7 @@ export class ItemService {
     const dbItem = await Item.findOne<ItemAttributes>(id)
     if (!dbItem) {
       return false
-    } else if (isTPItem(dbItem)) {
+    } else if (dbItem.collection_id) {
```

Comment:
Removing `isTPItem` validation, this segment will validate if an `eth_address` is the manager or owner of the `Item`